server/models/Books.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose')
+mongoose.Promise = global.Promise
+
+const Schema = mongoose.Schema
+
+const novelSchema = new mongoose.Schema(
```

Comment:
What did you think about renaming this to `novelChapterSchema` in the end ? :)

Reply:
Thanks, this is now changed.